packages/webcomponents/src/components/business-form/business-form-stepped.tsx

```diff
@@ -25,6 +25,9 @@ export class BusinessFormStepped {
   @State() totalSteps: number = 4;
   @State() serverError: boolean = false;
   @State() errorMessage: string = '';
+  @Event() clickEvent: EventEmitter<any>;
+  @Event() submitted: EventEmitter<any>;
```

Comment:
I think we should type this so that the emitted event is consistent with other things that do this such as `CardForm`: https://github.com/justifi-tech/web-component-library/blob/main/packages/webcomponents/src/components/card-form/card-form.tsx#L45